ckanext/datagovtheme/helpers.py

```diff
@@ -526,3 +526,6 @@ def get_bureau_info(bureau_code):
         bureau_info['logo'] = None
 
     return bureau_info
+
+def ckan_version_28():
```

Comment:
Sorry I didn't catch this earlier. Can we change this function to be more semantic? e.g. it's not really the version of CKAN that matters, it matters if we're using a CKAN version that supports bootstrap 2 vs 3. So `is_bootstrap2` might make more sense, and the default is to assume bootstrap 3.

For example, when CKAN 2.9 comes out, you shouldn't have to add a new function ckan_version_29().